plugins/SlicerT/SlicerTWaveform.cpp

```diff
@@ -168,16 +173,30 @@ void SlicerTWaveform::drawEditorWaveform()
 
 	QPainter brush(&m_editorWaveform);
 	size_t startFrame = m_seekerStart * m_slicerTParent->m_originalSample.sampleSize();
-	size_t endFrame = m_seekerEnd * m_slicerTParent->m_originalSample.sampleSize();
+	size_t   endFrame = m_seekerEnd * m_slicerTParent->m_originalSample.sampleSize();
 
 	brush.setPen(s_waveformColor);
-	float zoomOffset = (m_editorHeight - m_zoomLevel * m_editorHeight) / 2;
+	long zoomOffset = (m_editorHeight - m_zoomLevel * m_editorHeight) / 2;
```

Comment:
Are you sure we should be converting this to `long`?

Reply:
The only use of this `zoomOffset` is to be passed into `sampRect` in `VisualizeParameters`. I can bring that line down and remove the variable entirely

Reply:
Up to you, but I wasn't suggesting to remove the variable: was just wondering if the data type change was necessary (was thinking maybe other parts of the code where expecting a floating point number for `zoomOffset`)